doc/images/.gitignore

```diff
@@ -1,4 +1,5 @@
 resource_management.svg
+ipsec_fsm.svg
```

Comment:
This was removed by PR #320. You may need to clean your local directory. This file no longer exists.